server/resp/src/main/java/org/infinispan/server/resp/commands/set/SRANDMEMBER.java

```diff
@@ -28,6 +29,10 @@ public CompletionStage<RespRequestHandler> perform(Resp3Handler handler,
          ChannelHandlerContext ctx,
          List<byte[]> arguments) {
       final Long count = (arguments.size() <= 1) ? 1 : ArgumentUtils.toLong(arguments.get(1));
+      if (count == Long.MIN_VALUE) {
```

Comment:
I wonder if this is enough, user can still pass in even lower value then Long.MIN_VALUE. Even though then it'd cause a different error?

Reply:
Lower values are handled by `toLong()` as they are not Long values.

MIN_VALUE=-9223372036854775808 is a kind of corner case since it's actually a long, but its opposite 9223372036854775808 is not, so it can't be used to count how many values return

error msg seems good for lower:
redis
```
127.0.0.1:6379> srandmember myset -92233720368547758082
(error) ERR value is not an integer or out of range
```
infinispan
```
127.0.0.1:6379> srandmember myset -92233720368547758082
(error) ERR value is not an integer or out of range
```